pkg/utils/helmrepoindex/interface.go

```diff
@@ -0,0 +1,20 @@
+package helmrepoindex
+
+import "time"
+
+type VersionInterface interface {
```

Comment:
Why defining a interface inside here, I think we'd better move it to `simple/client/openpitrix`

Reply:
Definitely, we need to move helmrepoindex and hemlwrapper to `simple/client/openpitrix`